substrate/client/chain-spec/src/genesis_config_builder.rs

```diff
@@ -134,7 +146,7 @@ mod tests {
 	#[test]
 	fn get_default_config_works() {
 		let config =
-			GenesisConfigBuilderRuntimeCaller::new(substrate_test_runtime::wasm_binary_unwrap())
+			<GenesisConfigBuilderRuntimeCaller>::new(substrate_test_runtime::wasm_binary_unwrap())
```

Comment:
```suggestion
			GenesisConfigBuilderRuntimeCaller::new(substrate_test_runtime::wasm_binary_unwrap())
```

Or was this some rustc bug/requirement?

Reply:
compilation bug (for other file, but the message was the same):
```
  --> cumulus/parachains/integration-tests/emulated/common/src/constants.rs:66:20
   |
66 |     let mut storage = GenesisConfigBuilderRuntimeCaller::new(code)
   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type of the type parameter `HF` declared on the struct `GenesisConfigBuilderRuntimeCaller`
   |
help: consider specifying the generic argument
   |
66 |     let mut storage = GenesisConfigBuilderRuntimeCaller::<HF>(code)
```

Reply:
rustc :roll_eyes: